doc/configuration.rst

```diff
@@ -713,6 +713,8 @@ If you customize return codes, prefer to set them in range 128 to 255 to avoid
 conflicts. Return codes in range 0 to 127 are reserved for possible future use
 by Luigi contributors.
 
+If you run *luigi.build()/luigi.run()* programmatically, you can enable a detailed response of type :class:`~luigi.execution_summary.LuigiRunResult` by sending ``detailed_summary=True`` to these functions. This detailed response has an attribute(``status``) which is a status code that contains information about the progress. The list of possible status codes can be found in :class:`~luigi.execution_summary.LuigiStatusCode`.
```

Comment:
This looks good as far as we are concerned. SUCCESS_WITH_RETRY is the status we are looking for.

Reply:
🍪 Sounds good 😄